includes/handler.inc

```diff
@@ -182,7 +182,7 @@ function islandora_oai_object_response_xml($params) {
         $link_element = $dom->createElementNS($record->record_namespace, "{$record->record_prefix}:{$field}", $object_url);
       }
       else {
-        $link_element = $dom->createElement("$field", $object_url);
+        $link_element = $dom->createElementNS('http://purl.org/dc/elements/1.1/', "$field", $object_url);
```

Comment:
@jordandukart  Your comment prompted me to think about the problem more.   I arrived on the solution using stock, unmodified code.

To get valid XML for Metadata Format: OAI_DC, one must set the Field value to just "identifier" (like the note says), and check the "Include record namespace" checkbox.